src/components/data/Modals.vue

```diff
@@ -0,0 +1,546 @@
+<template>
```

Comment:
Hvorfor er der så mange Modals? Er ideen med en template ikke at der er én eller måske to af dem, som bliver genbrugt alt efter behov?